packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -529,7 +538,7 @@ export class StripeHandler {
             paymentMethodCountry
           );
         }
-        if (paymentMethodCountry) {
+        if (!this.automaticTax && paymentMethodCountry) {
```

Comment:
Is there a ticket to capture removing tax IDs entirely after this Stripe Tax work is complete? Might be nice to reference it (or at least note this can be removed once Stripe Tax is enabled everywhere) here in the comments.

I imagine we can also remove the `setCustomerLocation` helpers and related code to update the customer's billing address (`customer.address`)?